envoy/BUILD_DEPS

```diff
@@ -34,9 +34,5 @@ envoy_cc_binary(
         "@envoy//source/common/router:config_utility_lib",
         "@envoy//source/exe:envoy_main_entry_lib",
         "@envoy//source/exe:envoy_common_lib",
-        "@istio_proxy//src/envoy/http/authn:filter_lib",
```

Comment:
Just out of curiosity, was it necessary to remove the Istio filters' deps? Now I think it's actually good to keep them, to make sure we don't break the compatibility with Istio.

Reply:
I may have hastened the removal, as these broke the build when I had not yet updated the ISTIO_SHA. However, I'd introduce these back in a separate PR as it takes quite a while to have the packer-ci-image & cilium-builder images recreated, sorry.

Reply:
No problem!